### PR TITLE
Jasons sphinx edits

### PR DESCRIPTION
**Pull request by moorepants**
I added numpydoc so that your docstrings are formatted nicely.

Sphinx still gives some warnings though.

### CODE REVIEW

File: docs/src/conf.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath('..'))
```
Comment (moorepants) [inline docs/src/conf.py:21]: This allows the docs to be built without installing the package first.

File: docs/src/conf.py [hunk 2]
```diff
@@ -57,9 +55,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.0.1'
+version = '0.1.0'
 # The full version, including alpha/beta/rc tags.
-release = '0.0.1'
+release = '0.1.0'
```
Comment (moorepants) [inline docs/src/conf.py:60]: Our first release will have a 0.1.0 release number.

**Comment by tarzzz** [comment]
Is it good to be merged?

**Comment by moorepants** [comment]
I'm requesting to pull against a branch you wrote, so it is you who gets to decided if this is good to be merged. If you don't like my changes, you should give feedback. If you do, then you can merge to you branch.

**Pull request merged**